vulture_os/portal/settings.py

```diff
@@ -114,10 +114,9 @@
 LANGUAGE_CODE = 'en-us'
 TIME_ZONE = 'UTC'
 USE_I18N = True
-USE_L10N = True
 USE_TZ = True
 
-STATIC_URL = '/static/'
+STATIC_URL = 'static/'
```

Comment:
is it really necessary?

Reply:
```
To allow serving a Django site on a subpath without changing the value of STATIC_URL,
the leading slash is removed from that setting (now 'static/')
```

Source : https://docs.djangoproject.com/en/dev/releases/4.0/#deprecated-features-4-0